### PR TITLE
Speech from atoms inside containers will now be properly made bold to…

### PR DESCRIPTION
**Pull request by DamianX**
… observers in vision range

Basically, before this PR: Be ghost, (Follow) X, X goes inside a closet, X speaks, the text is not bold despite being in your view.

I _think_ doing it this way is okay because ghost vision shouldn't be obstructed anyway, therefore a simple distance check should suffice, right?

This is my most nitpicky change as of yet.

### CODE REVIEW

**Comment by Exxion** [comment]
I'd recommend replacing `world.view` with `client ? client.view : world.view` or something so the bolding works as expected when zooming out/in

File: code/modules/mob/dead/observer/say.dm [hunk 1]
```diff
@@ -33,9 +33,8 @@
 
 	say_testing(src, "/mob/dead/observer/Hear(): source=[source], frequency=[speech.frequency], source_turf=[formatJumpTo(source_turf)]")
 
-	if (get_dist(source_turf, src) <= world.view) // If this isn't true, we can't be in view, so no need for costlier proc.
-		if (source_turf in view(src))
-			rendered_speech = "<B>[rendered_speech]</B>"
+	if (get_dist(source_turf, src) <= client ? client.view : world.view) // If this isn't true, we can't be in view, so no need for costlier proc.
```
Comment (unid15) [inline code/modules/mob/dead/observer/say.dm]: <= src.get_view_range()

Comment (Exxion) [reply code/modules/mob/dead/observer/say.dm]: Sorry, I didn't know about that proc

**Comment by Exxion** [comment]
I'm on mobile so pretend I added the QoL label

**Pull request merged**